<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>469</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>calc</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="5" rowspan="2">
     <widget class="QPushButton" name="EqualsButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="NumButton2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="ChangeSignButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="FactButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>x!</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="NumButton1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="NumButton3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="NumButton8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="NumButton9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="ExpButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>x^z</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="NumButton0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="AddButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="NumButton4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="LogButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>log(x)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #A9A9A9;
	border: 1px solid gray;
	color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="RootButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>√</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="SubButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="NumButton5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="NumButton6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="NumButton7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="ClearButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>A/C</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="MulButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pointButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="DivButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ffd700;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:hover{
	background-color: orange;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #ff7e00;
	border: 1px solid gray;
	padding:5px;
	border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QPushButton" name="HelpButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #DCDCDC;
	border: 1px solid gray;
	padding:5px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding:5px;
}
QPushButton:pressed{
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding:5px;
}</string>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

